src/Microsoft.AspNet.SignalR.Redis/RedisConnection.cs

```diff
@@ -17,7 +17,11 @@ public class RedisConnection : IRedisConnection
 
         public async Task ConnectAsync(string connectionString, TraceSource trace)
         {
-            _connection = await ConnectionMultiplexer.ConnectAsync(connectionString);
+            _connection = await ConnectionMultiplexer.ConnectAsync(connectionString, new TraceTextWriter("ConnectionMultiplexer: ", trace));
+            if(!_connection.IsConnected)
+            {
+                throw new InvalidOperationException("Failed to connect to Redis");
+            }
 
             _connection.ConnectionFailed += OnConnectionFailed;
```

Comment:
not related to this change but we seem to be subscribing to these events twice...